templates/include/telemetry/coreCommon.2.schema.json

```diff
@@ -152,4 +152,16 @@
     "type": "string",
     "description": "Added in bug 1612935."
   }
+},
+"enhancedSearchUsed": {
+  "type": "boolean",
+  "description": "Added in bug 1612935."
+},
+"enhancedSearchReady": {
+  "type": "boolean",
+  "description": "Added in bug 1612935."
+},
+"enhancedSearchVersion": {
+  "type": "string",
+  "description": "Added in bug 1612935."
```

Comment:
So in order to maintain backwards compatibility, should the `enhancedSearch` fields be left under `fennec` and in addition get added here to avoid missing columns errors?

Reply:
Yep, this is correct.